src/Command/Docker/Build.php

```diff
@@ -121,6 +123,11 @@ protected function configure()
                 null,
                 InputOption::VALUE_OPTIONAL,
                 'RabbitMQ version'
+            )->addOption(
+                self::OPTION_NODE,
+                null,
+                InputOption::VALUE_REQUIRED,
```

Comment:
All of the other services have optional versions. Why does Node.js need a required version unlike the others?

Reply:
I believe having the other arguments' values optional is actually a bug.

For PHP, passing just `--php` with no value doesn't do anything different than omitting the flag altogether.

For other services, (`--es`, `--rmq`) passing the flag with no value does *not* add the service as one may expect. The service is only added to `docker-compose.yml` if the flag include a value.

Reply:
Sounds good. Required makes sense then.